e2e-webdriver/src/page-objects/app.po.ts

```diff
@@ -0,0 +1,18 @@
+export class AppPage {
+
+  async navigateTo(): Promise<string> {
+    return browser.url('');
+  }
+
+  async getTitle(): Promise<string> {
```

Comment:
In der Dokumentation bei WDIO verfolgen sie einen anderen PO-Schnitt: Da wird normalerweise das Element als `get` rausgegeben (anstatt sein Textinhalt) und im Test werden dann Assertions wie `toHaveText()` verwendet.

Also hier beispielsweise
```ts
// app.po.ts
get title() {
  return $('app-root h1');
}
```
```ts
// my-test.e2e.ts
expect(appPage.title).toHaveText('GET YOUR ICECREAM');
```

Ich würde das hier auch so machen, damit wir möglichst nah an dem sind, wie die offizielle Doku es vorschlägt. Oder siehst du Gründe, die dagegen sprechen?

Reply:
Guter Vorschlag, das gefällt mir besser.